rotkehlchen/db/history_events.py

```diff
@@ -375,3 +376,25 @@ def get_value_stats(      # pylint: disable=no-self-use
             except DeserializationError as e:
                 log.debug(f'Failed to deserialize amount {row[1]}. {str(e)}')
         return usd_value, assets_amounts
+
+    def get_extra_data(self, event_identifier: int) -> Optional[dict[str, Any]]:
+        """Retrieve only the extra data information for a history event"""
+        with self.db.conn.read_ctx() as cursor:
+            cursor.execute(
+                'SELECT extra_data FROM history_events WHERE identifier=?',
+                (event_identifier,),
+            )
+            extra_data_row = cursor.fetchone()
+
+        extra_data = None
+        if extra_data_row is not None and (extra_data_value := extra_data_row[0]) is not None:
```

Comment:
Unless I'm wrong, the additional `(extra_data_value := extra_data_row[0]) is not None` check is not needed in this case.

Reply:
Yes, there was a test that triggered it because we store NULL and when you retrieve it is `(NULL,)`